libs/island-ui/core/src/lib/Navigation/Navigation.treat.ts

```diff
@@ -6,7 +6,9 @@ export const divider = style({
   height: 1,
 })
 
-export const root = style({})
+export const root = style({
+  transition: 'all 150ms',
```

Comment:
We should probably be more specific here regarding targeted transitions, `all` could have unwanted side effects and some performance impacts